JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -71,12 +71,13 @@ public async Task DispatchMessage(SQSMessage message, CancellationToken cancella
 
             var handlingSucceeded = false;
             Exception lastException = null;
+            IMessageContextAccessor ctxAx = new MessageContextAccessor();
```

Comment:
This will need wiring up into `IServiceResolver` at a later point so it's actually injectable.

Reply:
Commit wording is: temporary code in MessageDispatcher, intentionally ugly